loki/backend/fgen.py

```diff
@@ -1038,10 +1053,14 @@ def fgen(ir, style=None, depth=0, conservative=False):
     """
     Generate standardized Fortran code from one or many IR objects/trees.
     """
+    from loki.backend.fgencon import FortranCodegenConservative  # pylint: disable=import-outside-toplevel,cyclic-import
```

Comment:
If having this import as an outside top-level import, wouldn't it make more sense to put it within `if conservative`?